crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -57,13 +57,15 @@ pub fn transaction_request_to_typed(
         other,
     } = tx;
 
-    // Special case: OP-stack deposit tx
     if transaction_type == Some(0x7E) || has_optimism_fields(&other) {
         return Some(TypedTransactionRequest::Deposit(DepositTransactionRequest {
             from: from.unwrap_or_default(),
             source_hash: other.get_deserialized::<B256>("sourceHash")?.ok()?,
             kind: to.unwrap_or_default(),
-            mint: other.get_deserialized::<U256>("mint")?.ok()?,
+            mint: other
+                .get_deserialized::<Option<U128>>("mint")?
+                .ok()?
+                .map_or(Some(0), |mint| mint.try_into().ok()),
```

Comment:
@mattsse The reason of I open [issue](https://github.com/alloy-rs/alloy/issues/1121#event-13748219010) is U128 can be deserialized but u128 can't. I want use `get_deserialized::<Option<u128>>` directly rather than using `map_or`

Reply:
@mattsse Pls review